source/_integrations/linkedgo.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: LinkedGo
+description: Integrate LinkedGo devices
+ha_category:
+  - Binary sensor
+  - Climate
+  - Cover
+  - Energy
+  - Event
+  - Light
+  - Number
+  - Select
+  - Sensor
+  - Switch
+  - Text
+  - Update
+  - Valve
+ha_release: 2025.4
+ha_codeowners:
+  - '@balloob'
+  - '@bieniu'
+  - '@thecode'
+  - '@chemelli74'
+  - '@bdraco'
```

Comment:
I've added all the code owners of the Shelly integration. If anyone doesn't want to be a code owner for LinkedGo please let me know and I'll update the PR.

Reply:
These are synced with the parent integration automatically.

Reply:
Good to know, thanks.